apps/drupal/src/Plugin/Preprocessor/Preprocessors/Node/Node.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Drupal\particle\Plugin\Preprocessor\Preprocessors\Node;
+
+use Drupal\particle\Plugin\Preprocessor\Preprocessors\Base;
+
+/**
+ * Provides a base node preprocessor.
+ */
+class Node extends Base {
```

Comment:
Why doesn't this one have a `process()` function?

Reply:
It's not needed here since it'd just be empty. But if you wanted to only preprocess a Node, you'd add that via  `process` here.

Reply:
But Page has a preprocess function that isn't doing anything? Are you assuming that Page probably will need some processing and Node problem won't?

Reply:
Fair. I added it to Node. It's probably better that way as an example.

Reply:
I don't see it?

Reply:
Ack! Put it in the wrong spot. Updated!